tests/vcd_vapp.py

```diff
@@ -167,5 +168,55 @@ def test_111_instantiate_vapp_custom_disk_size(self):
         # cleanup
         self.test_100_delete_vapp()
 
+    def test_101_attach_disk_to_vm_in_vapp(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        v = org.get_vdc(self.config['vcd']['vdc'])
+        vdc = VDC(self.client, href=v.get('href'))
+        assert self.config['vcd']['vdc'] == vdc.get_resource().get('name')
+        vapp = vdc.get_vapp(self.config['vcd']['vapp'])
+        assert self.config['vcd']['vapp'] == vapp.get('name')
+        vapp_instance = VApp(self.client, resource=vapp)
+        disk = vdc.get_disk(self.config['vcd']['disk'])
+        vm_name = self.config['vcd']['vm']
+        result = vapp_instance.attach_disk_to_vm(disk.get('href'), disk.get('type'), disk.get('name'), vm_name)
+        task = self.client.get_task_monitor().wait_for_status(
+                            task=result,
+                            timeout=60,
+                            poll_frequency=2,
+                            fail_on_status=None,
+                            expected_target_statuses=[
+                                TaskStatus.SUCCESS,
+                                TaskStatus.ABORTED,
+                                TaskStatus.ERROR,
+                                TaskStatus.CANCELED],
+                            callback=None)
+        assert task.get('status') == TaskStatus.SUCCESS.value
+
+    def test_001_detach_disk_from_vm_in_vapp(self):
```

Comment:
Rename all the tests if the need be, so that they are lined up in increasing order. Btw all this is kinda just for show. There is no real benefit of appending binary numbers in front of the tests.